cmd/es/main.go

```diff
@@ -21,5 +21,9 @@ import (
 )
 
 func main() {
-	sdk.Serve(es.Specification, nil, esDestination.NewDestination)
+	sdk.Serve(sdk.Connector{
+		NewSpecification: es.Specification,
+		NewSource:        nil,
+		NewDestination:   esDestination.NewDestination,
+	})
```

Comment:
@hariso, @maha-hajja,
There is no https://github.com/ConduitIO/conduit-connector-sdk/tags version tagged with acceptance tests included thus `go.mod` requires `v0.2.1-0.20220608071937-511c321558fc`. Should this PR wait until the version is released?